scripts/util.py

```diff
@@ -207,25 +207,30 @@ def get_credentials() -> Optional[service_account.Credentials]:
 
 def create_client_options(api: ApiEndpoint = None) -> ClientOptions:
     """Create client options for cloud endpoints"""
-    ep = None
     ver = endpoint_version(api)
     ud = universe_domain()
-    if ud == DEFAULT_UNIVERSE_DOMAIN:
-        ud = None
+    co = ClientOptions()
```

Comment:
nit. `ClientOptions` appears to act as a simple "named tuple", I would recommend to not directly override object properties, instead call a constructor once and don't mutate it after that. E.g.

```py
options = {}
if ud != DEFAULT_UNIVERSE_DOMAIN:
   options["universe_domain"] = ud
if ver:
  options["api_endpoint"] = f"https://{api.value}.{ud}/{ver}/"
return ClientOptions(**options)
```

Reply:
Done